test/examples/examples.js

```diff
@@ -329,6 +329,11 @@ const cases = [
     bindings: { person: address('0x0000000000000000000000000000000000000001') },
     options: { to: '0x960b236A07cf122663c4303350609A66A7B288C0' }
   }, 'Send ETH to the sale at block 3723000 from 0x0000000000000000000000000000000000000001'],
+  [{
+    source: "Transaction with ID `txId` was sent to `self.getTransaction(txId): (uint64, uint256, uint256, uint64, address, [address], bool, uint64)`",
+    bindings: { txId: { type: 'uint256', value: 1 } },
+    options: { to: '0xf562B25Db6e707694ceC3A4908dC58fF6bDABa40' },
+  }, 'Transaction with ID 1 was sent to 0x52EC80600642CeddE9De1F570335481C348BE74C'],
```

Comment:
Would be nice to a few more tests with single returns, to make sure we have those cases covered:

- Implicit selection (without `()`)
- Implicit selection (with `()`)
- Explicit selection (with both `()` and `<>`)

It would also be nice to add tests that select something from the start and the end of the return types list, to make sure we're parsing that correctly.

Reply:
Case 1 is already covered by all other call tests. I can make an explicit one for this PR, but I don't think it will add much.

I'll add the other four cases.

Reply:
> Case 1 is already covered by all other call tests. I can make an explicit one for this PR, but I don't think it will add much.

Good point :).